task_sdk/src/airflow/sdk/definitions/mappedoperator.py

```diff
@@ -136,6 +137,22 @@ def ensure_xcomarg_return_value(arg: Any) -> None:
             ensure_xcomarg_return_value(v)
 
 
+def is_mappable_value(value: Any) -> TypeGuard[Collection]:
+    """
+    Whether a value can be used for task mapping.
+
+    We only allow collections with guaranteed ordering, but exclude character
+    sequences since that's usually not what users would expect to be mappable.
+
+    :meta private:
+    """
+    if not isinstance(value, (Sequence, dict)):
+        return False
+    if isinstance(value, (bytearray, bytes, str)):
+        return False
+    return True
```

Comment:
What about a set? That should be false too?

Reply:
We didn't before. I just copied this (in my next massive PR the other one will be deleted.)

Let me update the source (somewhere in TI.py I think) to use this version